simson/common/data_transformations.py

```diff
@@ -65,6 +65,51 @@ def extrapolate_stock(
     return StockArray(**dict(stocks))
 
 
+def scale_parameter_to_future(historic_parameter : NamedDimArray, scaler : NamedDimArray) -> NamedDimArray:
+    """Scale parameter to future."""
+    dims = historic_parameter.dims.union_with(scaler.dims)
```

Comment:
Did I understand it correctly that this function is extrapolating `historic_parameter` to the future based on it's historic relationship with `scaler`? If so, this feels very much like it should be a new class in data_extrapolations.py.

Reply:
Good idea, I implemented that now.